### PR TITLE
display alert or confirm if no Write or Control

### PR DESCRIPTION
**Pull request by bourgeoa**
@TallTed @timbl 
This PR does 2 things : 
- ask for confirmation if there is no Control access (no "Owners")
- display an alert if there is no Write access (no "Owners" and no "Editors") : the resource cannot be deleted and the ACL resource cannot be accessed.

This should avoid some locking out of resources. 
This in  relation to This PR issue #396 and issue solid/solidcommunity.net#39 :

### CODE REVIEW

**Comment by bourgeoa** [comment]
Finally test resolved.

File: src/acl/access-controller.ts [hunk 1]
```diff
@@ -226,26 +229,47 @@ export class AccessController {
...
+      // check acl for acl:Write alert and acl:Control confirm
+      const hasWrite = newAClGraph.any(undefined, ns.acl('mode'), ns.acl('Write'), this.targetACLDoc)
+      const hasControl = newAClGraph.any(undefined, ns.acl('mode'), ns.acl('Control'), this.targetACLDoc)
+      const user = currentUser()
+      const webId = user ? user.uri : 'user'
+      if (!hasWrite) {
+        alert('There is no "Write access" this is not allowed')
+        return reject(new Error('ACL file save rejected : no acl:Write'))
+      } else if (!(hasControl || confirm('There is no "owners access" this is a dangerous situation !!!,' +
+         `\nAs ${webId}\nyou may definitly lose access to the resources covered by this ACL !!!` +
```
Comment (TallTed) [inline src/acl/access-controller.ts:247]:
```suggestion
      } else if (!(hasControl || confirm('There is no "owner access" -- this is a dangerous situation !!!,' +
         `\n${webId},\nyou may lose access to the resources covered by this ACL !!!` +
```

**Comment by TallTed** [comment]
Not sure why I was tagged on this ... but there are a few minor edits I'd make.

**Comment by bourgeoa** [comment]
Thanks. I saw you where reviewing human messages.

**Pull request merged**